### ISSUE TITLE
Implement Continuous Integration

### ISSUE THREAD

**Issue by kytrinyx** [exercism/xpascal#3]
Implement a test suite that can run both locally and on Travis CI, that verifies that the example code passes the test suite for each exercise.

Things to consider:

If some tests are marked as skipped or pending in some way, then CI should remove these annotations so that all the tests run against the example, not just the first test.

If we need to rewrite the files in any way, we should copy the exercise to a tmp directory first, so that we don't accidentally end up committing changes that are just for continuous integration or testing purposes.

We should make it possible to test just one exercise at a time, because testing everything can take a while, and very often people are just working on one exercise and will only need to run the full test suite before committing the change.

**Comment by rpottsoh** [comment]
@kytrinyx does Travis do Windows?  @kotp and I are concerned that it might not.  Yes, I know, I made a punny.

**Comment by kytrinyx** [comment]
@rpottsoh I don't know. I don't think so. Will AppVeyor work?

**Comment by rpottsoh** [comment]
Good question.  I am not familiar with AppVeyor.   Will go read about now.

**Comment by rpottsoh** [comment]
At quick glance it looks like it provides a VM in which we would need to install Windows for Delphi to run on or is AppVeyor providing Windows too?  This looks promising.   Thanks for pointing it out.

**Comment by kotp** [comment]
Reference links, @rpottsoh ??

**Comment by rpottsoh** [comment]
AppVeyor [landing page](https://www.appveyor.com/)
[Documentation](https://www.appveyor.com/docs/)

**Comment by rpottsoh** [comment]
Just noticed file `appveyor.yml` sitting in xcsharp.  I'm assuming AppVeyor is already in use by some tracks.

**Comment by kytrinyx** [comment]
Yeah, C#, F#, and it's also turned on for VB.NET and (now) Delphi.

**Comment by rpottsoh** [comment]
@kotp how do we proceed?  I am out of my element here, but interested to watch, learn and help.

**Comment by kotp** [comment]
We can work it after your happy anniversary weekend!
